plugins/workspace-minimap/src/minimap.ts

```diff
@@ -44,6 +44,9 @@ export class Minimap {
    */
   constructor(workspace: Blockly.WorkspaceSvg) {
     this.primaryWorkspace = workspace;
+    this.minimapWorkspace = new Blockly.WorkspaceSvg(new Blockly.Options({}));
+    this.focusRegion = new FocusRegion(this.primaryWorkspace, this.minimapWorkspace);
+    this.minimapWrapper = null;
```

Comment:
No need to assign in the constructor if it is `null` by default. Please remove this.